field/koalabear/extensions/e2_amd64.go

```diff
@@ -0,0 +1,36 @@
+package extensions
+
+import (
+	fr "github.com/consensys/gnark-crypto/field/koalabear"
+)
+
+// q + r'.r = 1, i.e., qInvNeg = - q⁻¹ mod r
+// used for Montgomery reduction
+const qInvNeg uint64 = 725501752471715839
+
+// Field modulus q (Fr)
+const (
```

Comment:
not sure this is the right values

Reply:
I guess we can simply drop the AMD code for now

Reply:
sorry mistakenly pushed wrong asm code. Commit [e96f6bb](https://github.com/Consensys/gnark-crypto/pull/643/commits/e96f6bb4164af490a115a78148987a1db3248dbc) fixed it.